include/Metabox/Posts/Category.php

```diff
@@ -92,7 +92,16 @@ public function move( $options ) {
 		foreach ( $posts as $post ) {
 			$current_cats = array_diff( wp_get_post_categories( $post->ID ), array( $options['old_cat'] ) );
 
-			if ( - 1 !== $options['new_cat'] ) {
+			if ( -1 !== $options['new_cat'] ) {
+				if ( $options['overwrite'] ) {
+					// Remove old categories.
+					$current_cats = array( $options['new_cat'] );
+				} else {
+					// Add to existing categories.
+					$current_cats[] = $options['new_cat'];
+				}
+			}else if( -1 == $options['new_cat'] ){
+				$options['new_cat'] = absint( $options['new_cat'] );
```

Comment:
`$options['new_cat']` is -1. When you call `absint` on this, then it will become +1. This code/logic is wrong.

Reply:
Fixed